front/src/routes/integration/all/tuya/commons/deviceHelpers.js

```
@@ -0,0 +1,224 @@
+    features.forEach(feature => {
+      const parts = (feature.external_id || '').split(':');
+      const code = parts.length >= 2 ? parts[parts.length - 1] : null;
+      addKnownCode(code);
+    });
+  }
+  const services = Array.isArray(device && device.thing_model && device.thing_model.services)
+    ? device.thing_model.services
+    : [];
+  services.forEach(service => {
+    const properties = Array.isArray(service && service.properties) ? service.properties : [];
+    properties.forEach(property => addKnownCode(property && property.code));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)